sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_administration_client.py

```diff
@@ -240,16 +193,15 @@ def create_anomaly_alert_configuration(
                 :caption: Create an anomaly alert configuration
         """
 
-        cross_metrics_operator = kwargs.pop("cross_metrics_operator", None)
         response_headers = self._client.create_anomaly_alerting_configuration(  # type: ignore
             _AnomalyAlertingConfiguration(
```

Comment:
Isn't it alert_configuration._to_generated_patch()

Reply:
That will only work correctly for updating an anomaly alert configuration, not create. Are you saying to add a `to_generated()` here?

Reply:
What is missing in _to_generated_patch? I checked the code and it seemed to me same?

Reply:
`_to_generated_patch` is returning an AnomalyAlertConfigurationPatch rather than an AnomalyAlertConfiguration. I suppose it would work currently because the patch model and regular model are the same for an alert config. If this ever changed in some later version, using this call could break it.

Reply:
So maybe we should add a to_generated() method.